pulser-core/pulser/sequence/sequence.py

```diff
@@ -1632,6 +1632,33 @@ def draw(
     def _plot(self, **draw_options: bool) -> tuple[Figure | None, Figure]:
         return draw_sequence(self, **draw_options)
 
+    def _add_slm_pulse(self, pulse: Pulse, channel: str) -> None:
+        if (
+            self._in_ising
+            and self._slm_mask_dmm
+            and self.declared_channels[channel].addressing == "Global"
+            and isinstance(self.declared_channels[channel], DMM)
```

Comment:
This doesn't seem right... I think you probably meant the opposite, no?

Reply:
Yes indeed, it is to avoid having it used during modulate_dmm (althought I figured out later that it was not mandatory as the pulse added via this method are detuned_delays)